applications/RomApplication/python_scripts/petrov_galerkin_training_utility.py

```diff
@@ -35,6 +32,10 @@ def __init__(self, solver, custom_settings):
         self.basis_strategy = settings["basis_strategy"].GetString()
         self.include_phi = settings["include_phi"].GetBool()
         self.svd_truncation_tolerance = settings["svd_truncation_tolerance"].GetDouble()
+        self.rom_basis_output_name = custom_settings["rom_basis_output_name"].GetString()
+        self.rom_basis_output_folder = custom_settings["rom_basis_output_folder"].GetString()
+        self.rom_basis_output_folder = Path(self.rom_basis_output_folder)
+        self.rom_basis_output_name = Path(self.rom_basis_output_name)
```

Comment:
```suggestion
        self.rom_basis_output_name = Path(custom_settings["rom_basis_output_name"].GetString())
        self.rom_basis_output_folder = Path(custom_settings["rom_basis_output_folder"].GetString())
```
I think it is more clear this way.